home/utils/image.py

```diff
@@ -6,7 +6,7 @@
 from django.core.files.base import ContentFile
 
 
-def convert_svg_to_png_bytes(svg_file_path, fill_color=None, stroke_color=None, scale=100):
+def convert_svg_to_png_bytes(svg_file_path, fill_color=None, stroke_color=None, width=16):
```

Comment:
update the usage of this parameter in the function

Reply:
This should be 32 rather than 16 right?

Reply:
Yes. I believe the line highlighted in green is the current state of the code, not the suggested change.

Reply:
Sorry for the delay, I've updated this.